tests/helm/metrics_test.go

```diff
@@ -217,22 +220,9 @@ sumologic:
 	defaultExporter := otelConfig.Exporters["sumologic/default"]
 	assert.Equal(t, "prometheus", defaultExporter.MetricFormat)
 	assert.Equal(t, "${SUMO_ENDPOINT_DEFAULT_METRICS_SOURCE}", defaultExporter.Endpoint)
-	assert.Contains(t, otelConfig.Processors, "routing")
-	assert.Contains(t, otelConfig.Service.Pipelines.Metrics.Processors, "routing")
-	assert.Equal(
-		t,
-		[]string{
-			"sumologic/default",
-			"sumologic/apiserver",
-			"sumologic/control_plane",
-			"sumologic/controller",
-			"sumologic/kubelet",
-			"sumologic/node",
-			"sumologic/scheduler",
-			"sumologic/state",
-		},
-		otelConfig.Service.Pipelines.Metrics.Exporters,
-	)
+	assert.Contains(t, otelConfig.Connectors, "routing/default")
```

Comment:
Should there be an assertion here to ensure all the expected pipelines are present?